app/scripts/models/user.js

```diff
@@ -372,7 +373,10 @@ define(function (require, exports, module) {
       var self = this;
       return account.signIn(password, relier, options)
         .then(function () {
-          if (! account.get('verified')) {
+          var isSignUp =
```

Comment:
Consider setting the default reason in fxa-client.js or account.js so that user.js does not have to worry about backwards compatibility and checking whether `verificationReason` is set.